app/Http/Controllers/Admin/BusinessController.php

```diff
@@ -91,7 +91,16 @@ public function delete($id, BusinessRepository $businessRepository, CommandBus $
         if (!$business) {
             return response('', 404);
         }
-        $this->authorize('update', $business);
+
+        if ($business->getPublishingStatus() === PublishingStatus::DRAFT ||
+            $business->getPublishingStatus() === PublishingStatus::READY_FOR_REVIEW) {
+            // Not live on the site yet.
+            $this->authorize('update', $business);
+        } else {
+            // Live on the site, and therefore require higher permissions.
+            $this->authorize('publish', $business);
+        }
+
```

Comment:
This logic is better - though wonder if it could be factored out in to its own 'delete' policy?

Reply:
It could.  It depends if there is a realistic prospect of someone having publish but not delete permissions.  Let's see where your table leads us.

Reply:
Let's leave this as is for now then.